examples/webgl_loader_svg.html

```diff
@@ -161,9 +161,9 @@
 					group.position.y = 70;
 					group.scale.y *= - 1;
 
-					for ( var i = 0; i < paths.length; i ++ ) {
+					for ( key in paths['paths'] ) {
```

Comment:
This seems over complicated. You should be able to keep the previous code and just add a these lines 
```js
loader.load( url, function ( data ) {
```
```js
var paths = data.paths;

for ( var i = 0; i < paths.length; i ++ ) {
```